### DISCUSSION TITLE
Review of Translator

### DISCUSSION THREAD

**Discussion by AlexeyMatveev686** [ONLYOFFICE/onlyoffice.github.io#240] [Polls]
<img src="https://raw.githubusercontent.com/ONLYOFFICE/onlyoffice.github.io/master/discussions-icons/translator.svg">

**Comment by g4t03ts** [comment]
Mantab, very very nice ...

**Comment by bcai01** [comment]
When translating multi-line text copied from a PDF, it should not be forced into a single merged line before translation.

**Reply by Misokados** [reply]
Hello,
I've checked the latest plugin version (v.1.0.9.). All things are OK. Please re-check the situation.

**Comment by regs01** [comment]
Consumes huge amount of RAM, when trying to translate several thousand cells.
<img width="548" height="78" alt="Taskmgr_2025-09-16_14-44-47" src="https://github.com/user-attachments/assets/56c1815a-67e0-451e-ab77-aa41d467519a" />

**Reply by Misokados** [reply]
Hello,
If it's possible please provide us with a test file and detailed scenario. We will check it out.
